lib/viewmodels/users_ranking_view_model.dart

```diff
@@ -64,6 +68,24 @@ class UsersRankingViewModel extends AutoDisposeAsyncNotifier<RankingDetails> {
       userRankElementDetails: currentUserRankingDetails,
     );
 
+    // Refresh the centauri points of the top users
+    for (final topUsers in topUsersFromDb) {
+      ref
+          .read(
+            userCentauriPointsViewModelProvider(topUsers.uid).notifier,
+          )
+          .refreshWithCentauriPointsNumber(
+            topUsers.data.centauriPoints,
+          );
+    }
+
+    //Refresh the centauri points of the current user
+    ref
+        .read(
+          userCentauriPointsViewModelProvider(currentUser.uid).notifier,
+        )
+        .refreshWithCentauriPointsNumber(currentUserData.centauriPoints);
```

Comment:
The following allows to slightly decrease code repetition:
```
    // Refresh the centauri points of the top users and the current user
    for (final user in topUsersFromDb + [currentUser]) {
      ref
          .read(
            userCentauriPointsViewModelProvider(user.uid).notifier,
          )
          .refreshWithCentauriPointsNumber(
            user.data.centauriPoints,
          );
    }
```

This is not a GitHub suggestion since I think it requires removing unused variables.

Reply:
You are right !

Added in: [3347ba7](https://github.com/ProximaEPFL/proxima/pull/348/commits/3347ba7256a1066d388c2e66814259b6399a4be8)

Thank you !